inference/core/workflows/core_steps/classical_cv/distance_measurement/v1.py

```diff
@@ -0,0 +1,345 @@
+from typing import Dict, List, Literal, Optional, Tuple, Type, Union
```

Comment:
If I read correctly, the block does not calculate the distance from centers of bboxes - and this is probably what would be most beneficial

Reply:
i think both are beneficial yes, with center of bboxes less priority. for john deere they are looking for vertical distance between 2 bounding boxes from edges

Reply:
@ryanjball please avoid giving out client names